doc/sphinx/run.rst

```diff
@@ -225,7 +225,15 @@ where :math:`\Delta E=E_\mathrm{new}-E_\mathrm{old}` is the change in potential
 and :math:`\beta=1/k_\mathrm{B}T`. 
 The extent of reaction, :math:`\xi=1` for the forward, and
 :math:`\xi=-1` for the backward direction. 
-The parameter :math:`\Gamma` proportional to the reaction constant. It is defined as
+The parameter :math:`\Gamma`  is defined as
+
+.. math::
+
+   \Gamma = \prod_i \Bigl(\frac{q_i}{\Lambda_i} \Bigr)^{\bar\nu} 
+
+where :math:`q_i` and :math:`\Lambda_i` are the single-particle partition function and
+thermal wavelength of species :math:`i`. In the absence of intermolecular interactions
+(ideal gas or ideal solution), :math:`\Gamma` proportional to the concentration-based reaction constant
```

Comment:
Gamma is the concentration based reaction constant in units of 1/(sigma^3)^nu_bar, if no interactions are present. Proportional is too weak for my taste.

Reply:
I agree but I will not specify the units because these are defined by the simulation script. The units have to be consistent with other units in the simulation script. Can be mol/L or 1/sigma^3 or anything else.

Reply:
It all depends on the unit of the volume that is provided that is correct. However for the case that the user does not provide a volume (which is standard and V=box_l_x*box_l_y*box_l_z, V is in units of sigma^3) and then as a standard the unit of gamma is 1/(sigma^3)^nubabr

Reply:
The sentence before says: "Note that the dimension of :math:`\Gamma` is :math:`V^{\bar\nu}`, therefore its units must be consistent with the units in which Espresso measures the box volume, i.e. :math:`\sigma^3`." Isn't this precisely the same thing as you wrote, just spelled differently?